### PR TITLE
Adds the MF Doom Mask.

### PR DESCRIPTION
**Pull request by ArcaneMusic**
## About The Pull Request

This PR adds a new mask and voice changer to the game, based on the Mad Villainy himself.
Ironically enough, there's enough legal distance between the rapper and the Marvel Character that we have plausible deniability either way.
Wearing the mask performs some text replacement on some of your speech.
Paid code, all credit goes to Serpico on discord.
It is available in the Costume Vendor as a premium item.

## Why It's Good For The Game

Guy's a great rapper, Marvel's Doc Doom is sick, and I'm being paid to do this anyway.

## Changelog
:cl: Serpico
add: Adds a new mask to the game, the Mad Villain mask to the costumedrobe.
/:cl:

### CODE REVIEW

File: strings/DOOMspeak.json [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+		"husk": "little charred body smelted like chicken",
+		"me": "the MADVILLIAN",
+		"murderboning": "dashing in a beat-rhyme crime spree",
+		"my name": "the mad name",
+		"retard": "average bug-a-boo",
+		"shitsec": "the real thugs, killers and gangsters",
+		"shot": "A slug through the vest, shot in the street",
+		"speak": "Whip up a slice of nice verse pie",
+		"test": "mic check",
+		"wizard": "Villian"
```
Comment (LemonInTheDark) [inline strings/DOOMspeak.json:23]: Some of these are song lyrics, spose we've gone further in the past

File: code/modules/clothing/masks/miscellaneous.dm [hunk 2]
```diff
@@ -184,3 +184,36 @@
 			message = replacetextEx(message,regex(capitalize(key),"g"), "[capitalize(value)]")
 			message = replacetextEx(message,regex(key,"g"), "[value]")
 	speech_args[SPEECH_MESSAGE] = trim(message)
+
+/obj/item/clothing/mask/Madvilliansmask
```
Comment (LemonInTheDark) [inline code/modules/clothing/masks/miscellaneous.dm:188]: No capitalized typepaths

File: code/modules/clothing/masks/miscellaneous.dm [hunk 3]
```diff
@@ -184,3 +184,36 @@
...
+
+/obj/item/clothing/mask/Madvilliansmask/dropped(mob/M)
+	. = ..()
+	UnregisterSignal(M, COMSIG_MOB_SAY)
+
+/obj/item/clothing/mask/Madvilliansmask/handle_speech(datum/source, list/speech_args)
+	var/message = speech_args[SPEECH_MESSAGE]
+	if(message[1] != "*")
+		message = " [message]"
+		var/list/DOOMspeak_words = strings("DOOMspeak.json", "DOOMspeak")
```
Comment (LemonInTheDark) [inline code/modules/clothing/masks/miscellaneous.dm:206]: This should be static

**Comment by ArcaneMusic** [comment]
Closing for cringe

**Comment by wesoda25** [comment]
Did you get your 5 dollars?

File: code/modules/clothing/masks/miscellaneous.dm (hunk 3 above)
Comment (LemonInTheDark) [reply code/modules/clothing/masks/miscellaneous.dm:206]: woop it's cached, nvm

**Comment by Pugie** [comment]
reopen

**Pull request closed without merging**